src/cryptojwt/jwt.py

```diff
@@ -373,11 +394,10 @@ def unpack(self, token, timestamp=None):
         if self.msg_cls:
             _msg_cls = self.msg_cls
         else:
-            try:
-                # try to find a issuer specific message class
-                _msg_cls = self.iss2msg_cls[_info["iss"]]
-            except KeyError:
-                _msg_cls = None
+            # try to find an issuer specific message class
+            _msg_cls = self.iss2msg_cls.get(_info["iss"])
```

Comment:
if we are not sure that _info may not have 'iss' we should add an if test before